internal/settings/settings.go

```diff
@@ -56,7 +58,7 @@ const (
 	DefaultGatewayConfig           = "/etc/otel/collector/gateway_config.yaml"
 	DefaultOTLPLinuxConfig         = "/etc/otel/collector/otlp_config_linux.yaml"
 	DefaultConfigDir               = "/etc/otel/collector/config.d"
-	DefaultMemoryBallastPercentage = 33
+	DefaultMemoryBallastPercentage = 10
```

Comment:
Is there any reason to keep this var and `setMemoryBallast` around anymore?

Reply:
backward compatibility, eventually it will go away. Look above to our other thread.